src/apps/Registration.js

```diff
@@ -56,7 +57,15 @@ const RegistrationComponent = (props) => {
                         .auth()
                         .createUserWithEmailAndPassword(email, pass)
                         .then((user) => {
-                            alert("This is where Yashika works her magick");
+                            writeData();
+                            const docRef = firebase
+                                .firestore()
+                                .collection("Users")
+                                .doc();
```

Comment:
Instead of creating a random document, we need to place it somewhere deterministic so that we'll be able to access it later. Firebase provides a `uid` so that we can do this. You can find this in [the documentation](https://firebase.google.com/docs/reference/js/firebase.auth.Auth#createuserwithemailandpassword). You can access it as `user.uid` once you make the `{user}` change I mentioned above.
```suggestion
                                .doc(user.uid);
```